core/common/src/test/java/alluxio/resource/ResourcePoolTest.java

```diff
@@ -69,9 +69,10 @@ protected Integer createNewResource() {
 
   /**
    * Tests the normal acquiration of resource pools.
+   * @throws Exception
```

Comment:
NIT: for test cases, we don't javadoc exceptions. so you don't need to put ` * @throws Exception` here